holidayextras/plugin-tapestry
# Node bump

#### What does this PR do? (please provide any background)

[WEB-10000](https://hxshortbreaks.atlassian.net/browse/WEB-10000)

due to an issue with the mysql package tapestry-exodus uses when running under the-works on node 6.9.4 (https://github.com/holidayextras/the-works/pull/382) i have updated tapestry-exodus, to carry up the latest change i have done the same on plugin-tapestry

#### What tests does this PR have?
none

#### How can this be tested?

pull down https://github.com/holidayextras/the-works/pull/382 (currently link via the package.json) and boot up the-wall and theatreBlueprint, none of the site's functionality should of changed
 
#### Any tech debt?
fixed existing

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
